src/main/java/unibook/logic/commands/EditCommand.java

```diff
@@ -69,31 +154,25 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
+
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
```

Comment:
Missing bidirectionality. Set the modules this person is associated with similar to tags.